### PR TITLE
First pr

### PR DESCRIPTION
**Pull request by MAHANTH-wq**
I have added my profile.

### CODE REVIEW

**Review by bytesagar — COMMENTED** [review]
Thank you @MAHANTH-wq  for this PR. But we cannot merge this because, iniside yourUsername.md file you have missed image extension. Please add it and push new changes. 😊

File: src/contributors/MAHANTH-wq.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+
+ name: "MAHANTH"
+ username: "MAHANTH-wq"
+ image: "MAHANTH-wq"    // this image name should be the actual name of image you uploaded
```
Comment (coderosh-zz) [inline src/contributors/MAHANTH-wq.md]: Your image has extension of ".png". So, change "MAHANTH-wq" to "MAHANTH-wq.png"

Comment (anishghimire603) [reply src/contributors/MAHANTH-wq.md:5]: Yeah, @MAHANTH-wq Thanks for your contribution small changes need to be made in our PR, just change the extension of your image inside `src/contributors/MAHANTH-wq.md `  from  `image: "MAHANTH-wq" ` to `image: "MAHANTH-wq.png"` and you can remove the comment as well

File: src/contributors/MAHANTH-wq.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+ facebook: "mahanth"
+---
```
Comment (anishghimire603) [inline src/contributors/MAHANTH-wq.md:7]: You can add about yourself in line `8`

**Comment by MAHANTH-wq** [comment]
Added extension to the image

File: src/contributors/MAHANTH-wq.md (hunk 1 above)
Comment (MAHANTH-wq) [reply src/contributors/MAHANTH-wq.md]: okay

**Comment by bytesagar** [comment]
Thank you @MAHANTH-wq  for this small contribution

**Pull request merged**